WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsPresentingViewController.swift

```diff
@@ -3,18 +3,32 @@ import UIKit
 
 final class CardReaderSettingsPresentingViewController: UIViewController {
 
+    @IBOutlet weak var temporaryLabel: UILabel!
+
     /// An array of viewModels and related view classes
-    private var viewModelsAndViews = [CardReaderSettingsViewModelAndView]()
+    private var viewModelsAndViews: CardReaderSettingsPrioritizedViewModelsProvider?
 
     /// Set our dependencies
-    func configure(viewModelsAndViews: [CardReaderSettingsViewModelAndView]) {
+    func configure(viewModelsAndViews: CardReaderSettingsPrioritizedViewModelsProvider) {
         self.viewModelsAndViews = viewModelsAndViews
+        self.viewModelsAndViews?.onPriorityChanged = onViewModelsPriorityChange
```

Comment:
@ctarda - I am a little concerned that this will make a strong reference preventing this view controller from being destroyed. I'll double check that but your input is especially welcome.

Reply:
I don't see any reference cycles here, at least not in plain sight. Let's keep an eye on this anyway, we can always refer back here if we notice something weird going on